flink-table-store-connector/src/test/java/org/apache/flink/table/store/connector/AggregationITCase.java

```diff
@@ -0,0 +1,81 @@
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.store.connector;
+
+import org.apache.flink.types.Row;
+import org.junit.Test;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+
+import static org.apache.flink.util.CollectionUtil.iteratorToList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * ITCase for partial update.
+ */
+public class AggregationITCase extends FileStoreTableITCase {
+
+    @Override
+    protected List<String> ddl() {
+        return Collections.singletonList("CREATE TABLE IF NOT EXISTS T3 (" + "a STRING," + "b INT," + "c INT ," + "PRIMARY KEY (a) NOT ENFORCED )" + " WITH ('merge-engine'='aggregation'     );");
+    }
+
+    @Test
+    public void testMergeInMemory() throws ExecutionException, InterruptedException {
+        bEnv.executeSql("INSERT INTO T3 VALUES " + "('pk1',1, 2), " + "('pk1',1, 2)").await();
+        List<Row> result = iteratorToList(bEnv.from("T3").execute().collect());
+        assertThat(result).containsExactlyInAnyOrder(Row.of("pk1", 2, 4));
+    }
+
+    @Test
+    public void testMergeRead() throws ExecutionException, InterruptedException {
+        bEnv.executeSql("INSERT INTO T3 VALUES ('pk1',1, 2)").await();
+        bEnv.executeSql("INSERT INTO T3 VALUES ('pk1',1, 4)").await();
+        bEnv.executeSql("INSERT INTO T3 VALUES ('pk1',2, 0)").await();
+        List<Row> result = iteratorToList(bEnv.from("T3").execute().collect());
+        assertThat(result).containsExactlyInAnyOrder(Row.of("pk1", 4, 6));
+    }
+
+
+    @Test
+    public void testMergeCompaction() throws ExecutionException, InterruptedException {
+        // Wait compaction
+        bEnv.executeSql("ALTER TABLE T3 SET ('commit.force-compact'='true')");
+
+        // key pk1
+        bEnv.executeSql("INSERT INTO T3 VALUES ('pk1', 3, 1)").await();
+        bEnv.executeSql("INSERT INTO T3 VALUES ('pk1', 4, 5)").await();
+        bEnv.executeSql("INSERT INTO T3 VALUES ('pk1', 4, 6)").await();
+
+        // key pk2
+        bEnv.executeSql("INSERT INTO T3 VALUES ('pk2', 6,7)").await();
+        bEnv.executeSql("INSERT INTO T3 VALUES ('pk2', 9,0)").await();
+        bEnv.executeSql("INSERT INTO T3 VALUES ('pk2', 4,4)").await();
+
+        List<Row> result = iteratorToList(bEnv.from("T3").execute().collect());
+        assertThat(result).containsExactlyInAnyOrder(Row.of("pk1",11,12), Row.of("pk2",19,11));
+    }
+
+//    @Test
```

Comment:
Usually, it's not appropriate to leave commented code. If it's useless, remove it instead.

Reply:
This test example comes from `flink-table-store-connector/src/test/java/org/apache/flink/table/store/connector/PartialUpdateITCase.java`
I don't know if this test makes sense for `flink-table-store-core/src/main/java/org/apache/flink/table/store/file/mergetree/compact/AggregationMergeFunction.java`,
do I need to keep this test .

Reply:
> Usually, it's not appropriate to leave commented code. If it's useless, remove it instead.
This test example comes from `flink-table-store-connector/src/test/java/org/apache/flink/table/store/connector/PartialUpdateITCase.java`
I don't know if this test makes sense for `flink-table-store-core/src/main/java/org/apache/flink/table/store/file/mergetree/compact/AggregationMergeFunction.java`,
do I need to keep this test .

Reply:
> Usually, it's not appropriate to leave commented code. If it's useless, remove it instead.

This test example comes from `flink-table-store-connector/src/test/java/org/apache/flink/table/store/connector/PartialUpdateITCase.java`
I don't know if this test makes sense for `flink-table-store-core/src/main/java/org/apache/flink/table/store/file/mergetree/compact/AggregationMergeFunction.java`,
do I need to keep this test .